core/bentley/package.json

```diff
@@ -35,7 +35,7 @@
   "devDependencies": {
     "@itwin/build-tools": "workspace:*",
     "@itwin/eslint-plugin": "workspace:*",
-    "@opentelemetry/api": "^1.0.4",
+    "@opentelemetry/api": "1.0.4",
```

Comment:
According to the linked issue,if we were accidentally using types from 1.1.0, why not just bump the dep to ^1.1.0?

Reply:
Because that would be a breaking change if someone was specifically using the 1.0.4 version of opentelemetry/api and they had the itwin/core-bentley package as a dependency.

Reply:
Talked with paulius, guess its nbd the way this is. Our usage of opentelemtery is beta, and we probably couldve gotten away with just updating the peer dep to ^1.1.0, and pin it for dev deps and not had any ts file changes, but its whatever, nbd.